stdlib/hashtbl.ml

```diff
@@ -583,20 +604,28 @@ let find_all h key =
 
 let rec replace_bucket key data = function
   | Empty ->
-      true
+      None
   | Cons ({key=k; next} as slot) ->
       if compare k key = 0
-      then (slot.key <- key; slot.data <- data; false)
+      then (
+        let old_data = slot.data in
+        slot.key <- key;
+        slot.data <- data;
+        Some old_data)
```

Comment:
For uniformity, this code should be formatted exactly the same as the one in 445.

Reply:
yes